src/piral-build.test.ts

```diff
@@ -7,63 +7,192 @@ runTests('piral-build', ({ test, setup }) => {
     await ctx.run(`npm i emojis-list@3.0.0`);
   });
 
+  test('release-standard-template', 'can produce a release build', ['build.piral'], async (ctx) => {
+    await ctx.run(`npx piral build --type release`);
+
+    await ctx.assertFiles({
+      'dist/release/index.html'(content: string) {
```

Comment:
we should check that only release was build, i.e., the `dist/emulator` output should not exist.